casadm/cas_lib.c

```diff
@@ -597,7 +597,7 @@ static int get_core_device(int cache_id, int core_id, struct core_device *core)
 
 	core->id = core_id;
 	core->cache_id = cache_id;
-	strncpy_s(core->path, sizeof(core->path), cmd_info.core_path_name,
+	strncpy_s(core->path, sizeof(core->path)-1, cmd_info.core_path_name,
```

Comment:
According to strncpy_s comment header the second param is:

dmax      - restricted maximum length of the resulting dest, including the null

... so adding -1 here doesn't seem to be correct.